cmd/deployctl/cmd/build.go

```diff
@@ -32,13 +32,19 @@ const (
 	NormalizeInterfaceMTUFilterArg   = "normalize-mtu"
 	NormalizeConsoleFilterArg        = "normalize-console"
 	MinimalConfigFilterArg           = "minimal-config"
+	AllowDRBDConfigArg               = "allow-drbd"
+	AllowAllFileSystemsArg           = "allow-all-fs"
```

Comment:
suggest to use full term "allow-all-filesystems"

Reply:
Updated